subsys/testsuite/Kconfig

```diff
@@ -144,4 +144,22 @@ config TEST_ARM_CORTEX_M
 	  Note that by default, when building with ARM_SECURE_FIRMWARE
 	  set, these exceptions are set to target the Non-Secure state.
 
+config TEST_NEGATIVE_FATAL_HANDLER
+	bool "Using a pre-defined fatal handler to deal with negative case"
+	depends on ZTEST
+	default n
+	help
+	  This option will leverage a pre-defined fatal error handler to deal
+	  with scenario which running the negative test cases. This should be
```

Comment:
It's better to change the words like:

with scenario, where negative test cases run.

Reply:
Thanks for correcting this. Got it fixed.